controller/test/test_refugee_controller.py

```diff
@@ -0,0 +1,96 @@
+import unittest
+from datetime import date
+from models.camp import Camp
+from models.refugee import Refugee
+from models.plan import Plan
+import controller.refugee_controller as r
```

Comment:
TO align with other PRs, this should be imported as rc

Reply:
fixed!